gerousiabot/utils.py

```diff
@@ -32,7 +32,7 @@ def setup_logger() -> logging.Logger:
     return logger
 
 
-def get_api_token(key) -> str:
+def get_env_variable(key) -> str:
```

Comment:
Generic name, I assume? Plans to use utils to grab more than just API key in future?

Reply:
exactly. We can use it to grab any value from the `.env` files